esmvaltool/diag_scripts/galytska23/select_variables_for_tigramite.py

```diff
@@ -89,6 +105,12 @@ def calculate_slp(dict_item):
     var = iris.load_cube(dict_item['filename'])
     # calculate hPa from Pa.
     var.data /= 100
+    if var.var_name == "pressure_ural":
+        var.var_name = 'Psl_Ural'
+    elif var.var_name == 'pressure_sib':
+        var.var_name = 'Psl_Sib'
+    elif var.var_name == 'pressure_aleut':
+        var.var_name = 'Psl_Aleut'
```

Comment:
Unfortunately this doesn't work because the `var.var_name` for the variables `pressure_ural`, `pressure_sib`, and `pressure_aleut ` is `psl`. Therefore it impacts the saved output. And thats why I also suggest keeping the next function `variable_cases` as is. 
```suggestion
```